src/node_zlib.cc

```diff
@@ -181,10 +181,16 @@ class ZCtx : public AsyncWrap {
       // sync version
       ctx->env()->PrintSyncTrace();
       Process(work_req);
+#if ENABLE_TTD_NODE
```

Comment:
No need for ENABLE_TTD_NODE as the macro TTD_NATIVE_BUFFER_ACCESS_NOTIFY is noop if ENABLE_TTD_NODE is not defined